Formula/n/node.rb

```diff
@@ -46,13 +46,6 @@ class Node < Formula
 
   fails_with gcc: "5"
 
-  # We track major/minor from upstream Node releases.
-  # We will accept *important* npm patch releases when necessary.
-  resource "npm" do
-    url "https://registry.npmjs.org/npm/-/npm-10.8.3.tgz"
-    sha256 "b7dc7eb48d7479b93668e913c7ad686ab2aa71c705d4a56b5323d1bffdba2972"
-  end
```

Comment:
This is the same version as what's included with Node 22.9.0: https://github.com/nodejs/node/commit/61047dd130

Reply:
So that's what `# We track major/minor from upstream Node releases.` refers to?

Is there any technical reason of **why** then omitting the default way Node will attempt to bundle NPM?

Reply:
If there any reasons at all, whnich IMO are valid if given to OS or compatibility reasons -- can a warn/info be added?

Like "Homebrew will manually bundle the targeted npm version for this Node.js version -- read more at XXX.XYZ"

Giving knowledge to the end-user is always fundamental IMO.

Reply:
> So that's what `# We track major/minor from upstream Node releases.` refers to?

Yes. It means that the major and minor version should be the same, and ideally the patch version is the same unless there are very strong reasons not for it to be (see line just below that).

> Is there any technical reason of **why** then omitting the default way Node will attempt to bundle NPM?

Not really sure -- it's been done this way since Homebrew/legacy-homebrew#28075, which is well before I was involved in Homebrew.

> If there any reasons at all, whnich IMO are valid if given to OS or compatibility reasons -- can a warn/info be added?
> 
> Like "Homebrew will manually bundle the targeted npm version for this Node.js version -- read more at XXX.XYZ"

We can add it to `caveats` whenever the version doesn't match from the one upstream, but I don't know of any instance when it didn't.